src/game/logic/ai/turretai.cpp

```diff
@@ -13,3 +13,651 @@
  *            LICENSE
  */
 #include "turretai.h"
+#include "aiupdate.h"
+#include "gamelogic.h"
+#include "partitionmanager.h"
+#include "randomvalue.h"
+#include "terrainlogic.h"
+
+StateReturnType Frame_To_Sleep_Time(unsigned int frame1, unsigned int frame2, unsigned int frame3, unsigned int frame4)
```

Comment:
Should this perhaps return `UpdateSleepTime` here?

See `UpdateModule::Frame_To_Sleep_Time` function for reference.

Reply:
No, this definitely returns StateReturnType.